src/PluginContext.ts

```diff
@@ -83,8 +82,7 @@ export function createPluginContext(
         ) {
           const result = await pl.resolveImport({ source, context });
           if (result) {
-            const fileUrl = new URL(result, `${pathToFileURL(importer)}`);
-            return { id: fileURLToPath(fileUrl) };
+            return { id: path.resolve(importer, result) };
```

Comment:
Why not `path.join` here?

Reply:
Yup, you're right. I'll correct it we don't need it to be resolved to the current directory, as `importer` is always absolute.